cibuildwheel/errors.py

```diff
@@ -0,0 +1,20 @@
+class FatalError(SystemExit):
```

Comment:
Btw, I couldn't decide whether to subclass SystemExit or Exception here, any strong opinions?

Reply:
On reflection, I'll change it to Exception to avoid the confusion around SystemExit's `code` attribute.
```suggestion
class FatalError(Exception):
```

Reply:
If it needs to behave like `SystemExit`, I'd inherit from `BaseException`. Have you considered this?

Reply:
(since it's raised in places where `sys.exit()` used to be, I'd expect this, I think)